<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="101900" started-at="2021-08-04T23:41:29 IST" finished-at="2021-08-04T23:43:11 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="101900" started-at="2021-08-04T23:41:29 IST" finished-at="2021-08-04T23:43:11 IST">
      <class name="com.automation.test.LoanCalculatorTest">
        <test-method status="PASS" signature="setUpSuite()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="setUpSuite" is-config="true" duration-ms="420" started-at="2021-08-04T23:41:28 IST" finished-at="2021-08-04T23:41:29 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpSuite -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="setUp" is-config="true" duration-ms="15326" started-at="2021-08-04T23:41:29 IST" finished-at="2021-08-04T23:41:44 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testCase_1()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="testCase_1" duration-ms="21671" started-at="2021-08-04T23:41:44 IST" finished-at="2021-08-04T23:42:06 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.automation.test.LoanCalculatorTest.testCase_1(LoanCalculatorTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase_1 -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="tearDownMethod" is-config="true" duration-ms="2628" started-at="2021-08-04T23:42:06 IST" finished-at="2021-08-04T23:42:08 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testCase_1 status=FAILURE method=LoanCalculatorTest.testCase_1()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="setUp" is-config="true" duration-ms="15928" started-at="2021-08-04T23:42:08 IST" finished-at="2021-08-04T23:42:24 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testCase_2()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="testCase_2" duration-ms="22258" started-at="2021-08-04T23:42:24 IST" finished-at="2021-08-04T23:42:47 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase_2 -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="tearDownMethod" is-config="true" duration-ms="2091" started-at="2021-08-04T23:42:47 IST" finished-at="2021-08-04T23:42:49 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testCase_2 status=SUCCESS method=LoanCalculatorTest.testCase_2()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="setUp" is-config="true" duration-ms="14558" started-at="2021-08-04T23:42:49 IST" finished-at="2021-08-04T23:43:03 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testCase_3()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="testCase_3" duration-ms="5307" started-at="2021-08-04T23:43:03 IST" finished-at="2021-08-04T23:43:09 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase_3 -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239]" name="tearDownMethod" is-config="true" duration-ms="2098" started-at="2021-08-04T23:43:09 IST" finished-at="2021-08-04T23:43:11 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testCase_3 status=SUCCESS method=LoanCalculatorTest.testCase_3()[pri:0, instance:com.automation.test.LoanCalculatorTest@3234e239] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
      </class> <!-- com.automation.test.LoanCalculatorTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
